sentry/sentry.conf.example.py

```diff
@@ -188,6 +188,13 @@ def get_internal_network():
 
 SENTRY_DIGESTS = "sentry.digests.backends.redis.RedisBackend"
 
+###################
+# Metrics Backend #
+###################
+
+SENTRY_RELEASE_HEALTH = "sentry.release_health.metrics.MetricsReleaseHealthBackend"
```

Comment:
should this live in https://github.com/getsentry/snuba/blob/master/snuba/settings/settings_self_hosted.py instead?